src/app/main/main.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from "@angular/core";
```

Comment:
do you need empty `main.scss`?
and again, why can't you use existing module/component instead main? What is specificity of main?